df.d_init.xml

```diff
@@ -137,6 +137,14 @@
         <static-array name='set_labor_lists' type-name='int16_t' count='2'/>
 
         <int32_t name='graze_coefficient' since='v0.40.13'/>
+        <int32_t name='temple_value_level_first' since='v0.47.01'/>
+        <int32_t name='temple_value_level_second' since='v0.47.01'/>
+        <int32_t name='priesthood_unit_count_first' since='v0.47.01'/>
+        <int32_t name='priesthood_unit_count_second' since='v0.47.01'/>
+        <int32_t name='guildhall_value_level_first' since='v0.47.01'/>
+        <int32_t name='guildhall_value_level_second' since='v0.47.01'/>
+        <int32_t name='guild_unit_count_first' since='v0.47.01'/>
+        <int32_t name='guild_unit_count_second' since='v0.47.01'/>
```

Comment:
Maybe these could be `static-array`s?

Reply:
Yes, static arrays look better (and probably match what the actual structures look like in DF).

Reply:
Actually, do you have any idea why these all have two values? For instance, if they're lower and upper bounds (which I don't think makes sense in this case, but maybe it does), using a struct or an index-enum might be nice.

Reply:
I know why, but not reasonable names. The first number is for the first petition, for the lower tier zone, while the second one is for the grander one. However, I don't know if the larger version has a distinct name, such as e.g. temple -> cathedral, and guild hall -> guild mansion.
(If I did, I would have replaced 1 and 2 in the original proposal with those).

Reply:
These should definitely be static-arrays, and their names should be plural to match what's actually in the file:
```
[PRIESTHOOD_UNIT_COUNTS:10:25]
[TEMPLE_VALUE_LEVELS:2000:10000]
[GUILD_UNIT_COUNTS:10:25]
[GUILDHALL_VALUE_LEVELS:2000:10000]